synapse/cortex.py

```diff
@@ -467,9 +469,13 @@ def pokeFormCount(self, form, valu):
             async for buid, valu in self.layer.iterFormRows(name):
 
                 count += 1
+                tcount += 1
 
-                if count % 10000 == 0:
+                if count % fairiter == 0:
                     await asyncio.sleep(0)
+                    # identity check for small integer
+                    if fairiter is 5 and tcount > 100000:
```

Comment:
a bit of a hyperoptimization, don't you think?

Reply:
a bit; but reduces the overall number of task switches we explicitly force for code that isn't going to be run very often (ideally)